cypress/integration/tests/administrator/custom-migration-targets/crud.test.ts

```diff
@@ -15,46 +15,125 @@ limitations under the License.
 */
 /// <reference types="cypress" />
 
-import { login, hasToBeSkipped } from "../../../../utils/utils";
-import { SEC } from "../../../types/constants";
+import { hasToBeSkipped, login } from "../../../../utils/utils";
+import { CredentialType, SEC, UserCredentials } from "../../../types/constants";
 import * as data from "../../../../utils/data_utils";
-import { CustomMigrationTarget } from "../../../models/administrator/custom-migration-targets/custom-migration-target";
+import {
+    RulesManualFields,
+    RulesRepositoryFields,
+    CustomMigrationTarget,
+    CustomRuleType,
+} from "../../../models/administrator/custom-migration-targets/custom-migration-target";
 import { CustomMigrationTargetView } from "../../../views/custom-migration-target.view";
+import { CredentialsSourceControlUsername } from "../../../models/administrator/credentials/credentialsSourceControlUsername";
 
 describe("Custom Migration Targets CRUD operations", { tags: ["@tier1", "@dc"] }, () => {
     beforeEach("Login", function () {
-        if (hasToBeSkipped("@tier1") && hasToBeSkipped("@dc")) return;
-
         login();
 
+        cy.fixture("custom-migration-targets").then(function (customMigrationTargets) {
+            this.customMigrationTargets = customMigrationTargets;
+        });
+
         cy.intercept("POST", "/hub/rulebundles*").as("postRule");
         cy.intercept("GET", "/hub/rulebundles*").as("getRule");
         cy.intercept("PUT", "/hub/rulebundles*/*").as("putRule");
         cy.intercept("DELETE", "/hub/rulebundles*/*").as("deleteRule");
     });
 
-    it("Custom Migration Targets CRUD", { tags: ["@tier1", "@dc"] }, function () {
+    it("Custom Migration Targets CRUD with rules uploaded manually", function () {
         CustomMigrationTarget.open();
+
+        const targetData = this.customMigrationTargets.manual_rules;
         const target = new CustomMigrationTarget(
             data.getRandomWord(8),
             data.getDescription(),
-            "img/cloud.png",
-            "xml/javax-package-custom-target.windup.xml"
+            targetData.image,
+            {
+                type: CustomRuleType.Manual,
```

Comment:
This will also come from targetData

Reply:
I've added this to the `utils` method but not to the json file as it is an enum created to avoid type mismatches.